src/HealthChecks.Uris/DependencyInjection/UrisHealthCheckBuilderExtensions.cs

```diff
@@ -71,6 +72,74 @@ public static IHealthChecksBuilder AddUrlGroup(this IHealthChecksBuilder builder
                 tags));
         }
         /// <summary>
+        /// Add a health check for single uri.
+        /// </summary>
+        /// <param name="builder">The <see cref="IHealthChecksBuilder"/>.</param>
+        /// <param name="uri">The uri to check.</param>
+        /// <param name="httpMethod">The http method to use on check.</param>
+        /// <param name="expectedContent">The content expected in the response body.</param>

```

Comment:
Can you explain in this comment that we read the content as plain string?

Reply:
Done. See method and param descriptions.